docs/sdks/stash/uid2-sdk-ref-mobile-gma.md

```diff
@@ -0,0 +1,99 @@
+# Developer Guide for UID2 GMA Plugin (iOS and Android)
+
+As of 2023-07-15 plugin version:
+* iOS V 0.2.0
+* Android V 0.4.0
+
+## Introduction
+
+The UID2 GMA Plugin is pre-release software and should be used as such.
+
+The UID2 GMA Plugin enables apps that use Google Mobile Ad (GMA) SDK to
+include UID2 data in their ad requests. This is done automatically, with
+no direct coding needed from the app developer, by:
+
+-   The Google Mobile Ad SDK
+
+-   The UID2 SDK
+
+-   The UID2 GMA Plugin
+
+## iOS 
+
+Refer to the following sections for iOS:
+
+-   Requirements
+
+-   Installation
+
```

Comment:
Should we have a _GitHub Open-Source Repository_ section like ios/android page to mention the source is on 
https://github.com/IABTechLab/uid2-ios-plugin-google-gma
?

Reply:
Done for external GMA file, thx. You'll see it in the GMA PR. Same wording as for Android and iOS:

## GitHub Open-Source Repository

This plugin is in the following open-source GitHub repository:

- [https://github.com/IABTechLab/uid2-ios-plugin-google-gma](https://github.com/IABTechLab/uid2-ios-plugin-google-gma)